Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -8995,6 +9073,7 @@ private void ValidationItem_ErrorsChanged(object sender, DataErrorsChangedEventA
         private void VerticalScrollBar_Scroll(object sender, ScrollEventArgs e)
         {
             ProcessVerticalScroll(e.ScrollEventType);
+            LoadMoreDataFromIncrementalItemsSource();
```

Comment:
Did you try to hook into `ValueChanged` of the ScrollBar rather than `Scroll` and `OnPointerWheelChanged`?

Reply:
I tried the ScrollBar approach initially but it didn't work the way I expected. Let me revisit that as it would be a bit cleaner.

Reply:
@MSRegisB I tried ScrollSlotIntoView but that is not it. I instead tried placing a break point in every method with the name scroll in it. When scrolling it only stops in one method, i.e. ScrollSlotsByHeight. I assume this is the common code path you are referring to...

I will move the trigger to this method, get rid of the reference to the vertical scrollbar and push a new commit.